admin/server/github.go

```diff
@@ -86,32 +87,82 @@ func (s *Server) GetGithubUserStatus(ctx context.Context, req *adminv1.GetGithub
 		return nil, fmt.Errorf("failed to update user: %w", err)
 	}
 
+	isUserAppInstalled := false
+	userInstallationPermission := ""
 	installation, _, err := s.admin.Github.AppClient().Apps.FindUserInstallation(ctx, user.GithubUsername)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get user installation: %w", err)
+		if !strings.Contains(err.Error(), "404") {
+			return nil, fmt.Errorf("failed to get user installation: %w", err)
+		}
+	} else {
+		isUserAppInstalled = true
+		userInstallationPermission = "read"
```

Comment:
Can we add an enum like `GithubPermission` in the proto for this instead of a string?